driver/src/main/scala/org/mongodb/scala/vault/package.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mongodb.scala
+
+/**
+ * This package contains the Key Vault API
+ *
+ * @since 2.7
+ */
+package object vault {
+
+  /**
+   * The Key vault.
+   *
+   * Used to create data encryption keys, and to explicitly encrypt and decrypt values when auto-encryption is not an option.
+   *
+   */
+  type ClientEncryption = com.mongodb.async.client.vault.ClientEncryption
```

Comment:
I thought this would have to wrap the callback-based methods in the async driver's class with methods that return Observables, similar to MongoCollection.scala.

Reply:
Good catch, I had a brain burp and missed that - new patch incoming.

Reply:
@jyemin updated the code